src/llama_stack_client/lib/cli/configure.py

```diff
@@ -26,8 +26,10 @@ def get_config():
 @click.command()
 @click.option("--host", type=str, help="Llama Stack distribution host")
 @click.option("--port", type=str, help="Llama Stack distribution port number")
+@click.option("--use-https", type=bool, default=False, help="Llama Stack distribution uses HTTPS")
 @click.option("--endpoint", type=str, help="Llama Stack distribution endpoint")
-def configure(host: str | None, port: str | None, endpoint: str | None):
+@click.option("--api-key", type=str, help="Llama Stack distribution API key")
+def configure(host: str | None, port: str | None, use_https: bool | None, endpoint: str | None, api_key: str | None):
```

Comment:
I think we can simplify this by just asking for a URL here instead of trying to cook up a URL using (is_https, host, port) shenanigans.

Reply:
I agree, it is little intense around that options. We can just use --endpoint then, it is exactly that, just a url.

Should I then I remove host, port and use_https options and have just leave endpoint url?